src/gui/optionsdlg.cpp

```diff
@@ -1182,6 +1206,9 @@ bool OptionsDialog::useAdditionDialog() const
 void OptionsDialog::enableApplyButton()
 {
     applyButton->setEnabled(true);
+
+    // Verify if the share action button must be enabled
+    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
```

Comment:
>That's because you haven't tried the PR, just read the code.

I also have not tested this, but I don't understand why the following code won't work?
```c++
connect(m_ui->checkMaxRatio, SIGNAL(toggled(bool)), this, SLOT(toggleComboRatioLimitAct()));
connect(m_ui->checkMaxSeedingMinutes, SIGNAL(toggled(bool)), this, SLOT(toggleComboRatioLimitAct()));

void OptionsDialog::toggleComboRatioLimitAct()
{
    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
}
```
Now you have it in the wrong place. If before no one noticed, that is no reason not to fix it.